querybook/webapp/components/DataDocGitHub/GitHubVersions.tsx

```diff
@@ -88,64 +119,71 @@ export const GitHubVersions: React.FunctionComponent<IProps> = ({
         );
     }
 
+    const commitListDOM = (
+        <div className="commit-list">
+            {commitVersions.map((version) => (
+                <CommitCard
+                    key={version.sha}
+                    version={version}
+                    onRestore={handleRestore}
+                    onCompare={toggleCompare.bind(null, version)}
+                />
+            ))}
+        </div>
+    );
+
+    const loadMoreButtonDOM = hasMore ? (
+        <AsyncButton onClick={fetchMore} className="load-more-button">
+            Load More
+        </AsyncButton>
+    ) : null;
+
+    const comparePanelDOM = (
+        <div
+            className={`compare-slide-out-panel ${
+                isCompareOpen ? 'open' : ''
+            } ${isFullScreen ? 'full-screen' : ''}`}
+        >
+            {selectedCommit && (
+                <div className="GitHubVersionsComparePanel">
+                    <div className="panel-header">
+                        <IconButton
+                            icon={isFullScreen ? 'Minimize2' : 'Maximize2'}
+                            onClick={toggleFullScreen}
+                            size={16}
+                            tooltip={
+                                isFullScreen
+                                    ? 'Exit Full Screen'
+                                    : 'Enter Full Screen'
+                            }
+                            tooltipPos="left"
+                        />
+                        <IconButton
+                            icon="X"
+                            onClick={() => toggleCompare()}
```

Comment:
simply  onClick={toggleCompare}

Reply:
type error complains because it detects that toggleCompare is not MouseEvent